src/xlsx/conversion/conversion_maps.ts

```diff
@@ -76,6 +79,15 @@ export const H_ALIGNMENT_CONVERSION_MAP: Record<XLSXHorizontalAlignment, Align>
   distributed: "center",
 };
 
+/** Conversion map Vertical Alignment in XLSX <=> Vertical Alignment in o_spreadsheet */
```

Comment:
```suggestion
/** Conversion map Vertical Alignment in XLSX => Vertical Alignment in o_spreadsheet */
```
Same in the export PR. The map is not 2-way (the proof is that you needed to create a different one for export 😅 )